README.md

```diff
@@ -72,3 +83,15 @@ We're using RSpec for testing. Run the test suite with rake spec. Tests for pull
 ## License
 
 `ruby-pipeine`` is free software released under the MIT License. See LICENSE for details.
+
+## Publishing a gem version
+
+This gem uses Github Packages Registry to store gems, and all builds get pushed to it.
+
+    Update lib/brokkr/version.rb with the new version
```

Comment:
`Update lib/ruby_pipeline/version.rb with the new version`?

Reply:
Closing

Reply:
no need actually 😄

Reply:
🙈 nice spot, cheers